README.md

```diff
@@ -224,6 +224,18 @@ Finally, let's make sure owership of protocol assets has been succsessfully tran
 yarn verify:ownership:transfer
 ```
 
+### ArbitrumFoundationVestingWallet Deployer
+
+#### Deploy
+- Set `ARB_URL` and `ARB_KEY` env vars
+- Set FoundationWalletDeploymentConfig variables (for target chain) in [config file](./scripts/foundation-wallet-deployment/config.ts)
+-  run ```yarn deploy:foundation-wallet ```
```

Comment:
```suggestion
- run ```yarn hardhat compile ```
- run ```yarn deploy:foundation-wallet ```
```

Reply:
`deploy:foundation-wallet` may use stale contract typechain
alternatively we can make `deploy:foundation-wallet` task to compile